services/device-registry-jdbc/src/test/java/org/eclipse/hono/deviceregistry/jdbc/impl/JdbcPostgresDeviceManagementSearchDevicesTest.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.deviceregistry.jdbc.impl;
+
+import java.nio.file.Path;
+
+
+class JdbcPostgresDeviceManagementSearchDevicesTest extends JdbcBasedDeviceManagementSearchDevicesTest {
```

Comment:
what is this class needed for?

Reply:
This class changes the DEFAULT_DATABASE_TYPE from H2 to POSTGRESQL and runs all the tests from JdbcBasedDeviceManagementSearchDevicesTest. We need this to run the tests against both types of JDBC DBs.

Reply:
The type of database being used when running the tests is determined by the `AbstractJdbcRegistryTest.databaseType` system property. The tests are being run for both H2 and Postgresql by means of the two configurations of the Maven Surefire plugin in `services/device-registry-jdbc/pom.xml`. FMPOV there is no need for this class.

Reply:
i remove the class